lib/filterx/expr-assign.c

```diff
@@ -42,7 +42,20 @@ _assign(FilterXBinaryOp *self, FilterXObject *value)
 static inline FilterXObject *
 _suppress_error(void)
 {
-  msg_debug("FILTERX null coalesce assignment supressing error", filterx_eval_format_last_error_tag());
+  if (trace_flag)
+    {
+      gint error_count = filterx_eval_get_error_count();
+      gchar buf[FILTERX_EVAL_ERROR_IDX_FMT_SIZE];
+
+      for (gint err_idx = 0; err_idx < error_count; err_idx++)
+        {
+          msg_debug("FilterX: null coalesce assignment suppressing error",
```

Comment:
is this trace or debug level? it used to be debug now it's actually trace due to the conditional.

Reply:
Nice catch, fixed.